openapi.yml

```diff
@@ -2590,30 +2590,13 @@ paths:
                       - some text
                       - some text
           description: Successful response - returns a single `Watchlist`.
-        '400':
+        4XX:
```

Comment:
Is this an option in the editor?
I didn't see how to do this in there

Reply:
You just have to manually enter 4XX, which counts as a wildcard in the openapi spec